.bazelci/presubmit.yml

```diff
@@ -3,7 +3,7 @@ aspects_flags: &aspects_flags
   - "--config=rustfmt"
   - "--config=clippy"
 min_rust_version_shell_commands: &min_rust_version_shell_commands
-  - sed -i 's|^rust_register_toolchains(|rust_register_toolchains(versions = ["1.59.0"],\n|' WORKSPACE.bazel
+  - sed -i 's|^rust_register_toolchains(|rust_register_toolchains(versions = ["1.62.0"],\n|' WORKSPACE.bazel
```

Comment:
What necessitates the MSRV bump? Something in our code, or something in a dependency we're picking up?

Reply:
At version 1.59.0, I get about a dozen of the following errors:
```
error[E0658]: `cfg(target_has_atomic)` is experimental and subject to change
   --> external/rules_rust_prost__log-0.4.19/src/lib.rs:349:7
    |
349 | #[cfg(target_has_atomic = "ptr")]
    |       ^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #32976 <https://github.com/rust-lang/rust/issues/32976> for more information
```

Bumping to 1.62.0 fixes this issue.